platforms/rpi/config.cmake

```diff
@@ -40,6 +40,7 @@ target_link_libraries(tangram
   /opt/vc/lib/libbrcmGLESv2.so
   /opt/vc/lib/libvchiq_arm.so
   /opt/vc/lib/libvcos.so
+  /usr/lib/arm-linux-gnueabihf/libatomic.so.1
```

Comment:
Using just `atomic` here seems to work on my pi. If it works on yours too, can you replace the path here with `atomic`? I'd rather not hard-code libraries where it isn't necessary.